core/vm/instructions.go

```diff
@@ -616,12 +616,10 @@ func opCreate(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]b
 
 	res, addr, returnGas, suberr := interpreter.evm.Create(scope.Contract, input, gas, bigVal)
 	// Push item on the stack based on the returned error. If the ruleset is
-	// homestead we must check for CodeStoreOutOfGasError (homestead only
+	// _ we must check for CodeStoreOutOfGasError (_ only
 	// rule) and treat as an error, if the ruleset is frontier we must
 	// ignore this error and pretend the operation was successful.
-	if interpreter.evm.chainRules.IsHomestead && suberr == ErrCodeStoreOutOfGas {
-		stackvalue.Clear()
-	} else if suberr != nil && suberr != ErrCodeStoreOutOfGas {
+	if suberr != nil {
```

Comment:
Did you mean to remove
&& suberr != ErrCodeStoreOutOfGas

Reply:
Nevermind, the ErrCodeStoreOutOfGas is a do not care. This code looks right. Leaving this for future reviewers.

Reply:
If the ruleset is homestead we must check for CodeStoreOutOfGasError